src/Populator/PropertyMappingPopulator.php

```diff
@@ -41,10 +41,16 @@ public function __construct(
     public function populate(object $target, object $source, object $ctx = null): void
     {
         try {
+            if (!empty($this->sourceProperty)) {
```

Comment:
`empty()` should be avoided.

```suggestion
            if ('' !== $this->sourceProperty) {
```

Reply:
This was new for me but I've seen that:
```
empty("0) is true
```
and that's of coure a no-go.